### ISSUE TITLE
Panel cannot detect F12 "dock side"

### ISSUE THREAD

**Issue by sanga000** [KC3Kai/KC3Kai#2212]
I think the title is pretty self explanatory. When I moved the F12 panel to the bottom, the game panel seem to think that it is still on the right and does not change its"orientation". This results in it being too long and I have to scroll to see all information on it. It should be noted that this only happens when I put the Chrome window at one half of the screen, but not when in any other position.

My Chrome version is up to date on a 64bit Windows.

### Screenshots (if applicable):
Responds to panel dock side normally
![error windowed](https://user-images.githubusercontent.com/26204896/30145225-85c89708-93c2-11e7-9f59-f1a7afdb8cd7.png)
Does not respond to panel dock side
![error side](https://user-images.githubusercontent.com/26204896/30145226-85cd7bc4-93c2-11e7-9921-14d7179b753d.png)

**Comment by mikenzb** [comment]
You are changing the dock side via the 3 vertical dot drop down menu? 
![capture](https://user-images.githubusercontent.com/5982780/30145317-b4c4c956-93e4-11e7-894b-a9c17e091f31.PNG)
KC3 can not automatically dock to a side if it detects there is not enough room.

**Comment by sanga000** [comment]
Yes, that's how I change it. I've been doing it the same way on my other computer and there's no problem, so I have no idea why this doesn't work.

**Comment by sanga000** [comment]
While browsing Git Hub I found someone else who seem to have the exact same problem, maybe this would make things clearer?

[https://github.com/KC3Kai/KC3Kai/issues/1950](url)

**Comment by Tedster59** [comment]
The problem here is that your screen is not wide enough for the panel to be in the wide mode when you have it docked to the side.  you can try zooming it out like your game is by holding ctrl and scrolling your mouse wheel on it.

**Comment by sanga000** [comment]
Alright, I see. It's working fine now, thank you.
